### ISSUE TITLE
Change profiles prompt

### ISSUE THREAD

**Issue by rothgar** [dutchcoders/cloudman#4]
Would love a way to switch which AWS profile I'm using from my $HOME/.aws/credentials file.

Even better if it could support multiple profiles/regions at once like https://github.com/disneystreaming/ssm-helpers

**Comment by nl5887** [comment]
Don't know if you are already aware of the `--profile` argument? I want to implement switching profiles pretty much like switching regions, but for now `--profile` could be sufficient. Will take a look at the ssm-helpers repo.

**Comment by rothgar** [comment]
I did see the profile option. In ssm-helpers we allowed for multiple profiles and regions to be used at the same time so a user could do something like `--profile prod,stage,dev` and `--region us-west-2,us-east-1,eu-central-1` and get a listing of all instances.

`awsls` also implemented this feature which is super handy for large customers https://github.com/jckuester/awsls

It would be great if the region select could be a toggle (space key selects multiple at the same time) but that's completely up to you if you want to support that.
